src/modules/flow/update/update.json

```diff
@@ -40,12 +40,6 @@
         "close": "common_close",
         "open": "check_open"
       },
-      "node_type": {
-        "access": [
-          "base"
-        ],
-        "data_type": "struct update_node_type"
```

Comment:
this is a hack. you are playing with something that clearly was not created to do. I really liked !!